File: backend/src/main/java/com/ll/hotel/domain/booking/booking/dto/BookingResponseSummary.java
package com.ll.hotel.domain.booking.booking.dto;

import com.ll.hotel.domain.booking.booking.type.BookingStatus;
import java.time.LocalDate;

public record BookingResponseSummary(
        long bookingId,
        long hotelId,
        long roomId,
        String hotelName,
        String roomName,
        String memberName,
        String thumbnailUrl,
        BookingStatus bookingStatus,
        int amount,
        LocalDate checkInDate,
        LocalDate checkOutDate
) {
}

File: backend/src/main/java/com/ll/hotel/domain/booking/booking/dto/BookingResponseSummary.kt
package com.ll.hotel.domain.booking.booking.dto

import com.ll.hotel.domain.booking.booking.type.BookingStatus
import java.time.LocalDate

data class BookingResponseSummary(
    val bookingId: Long,
    val hotelId: Long,
    val roomId: Long,
    val hotelName: String,
    val roomName: String,
    val memberName: String,
    val thumbnailUrl: String,
    val bookingStatus: BookingStatus,
    val amount: Int,
    val checkInDate: LocalDate,
    val checkOutDate: LocalDate
)